lis/surfacemodels/land/ac.7.2/ac72_readcrd.F90

```diff
@@ -204,16 +204,6 @@ subroutine AC72_readcrd()
      AC72_struc(n)%dt = AC72_struc(n)%ts
   enddo
 
-  ! reference height of forcings (T and q)
-  call ESMF_ConfigFindLabel(LIS_config, &
-       "AquaCrop.7.2 reference height of T and q:", rc = rc)
-  do n=1, LIS_rc%nnest
-     call ESMF_ConfigGetAttribute(LIS_config, AC72_struc(n)%refz_tq, rc=rc)
-     call LIS_verify(rc, &
-          "AquaCrop.7.2 reference height of T and q: "//&
-          "not defined")
-  enddo
-
   ! reference height of forcings (u and v)
   call ESMF_ConfigFindLabel(LIS_config, &
        "AquaCrop.7.2 reference height of u and v:", rc = rc)
```

Comment:
Could we just rename the text here to "Input forcing reference height..." and still keep it in the AC72_struc? I am worried that later some confusion about the reference height for AquaCrop would arise (while it is fixed 2m).